.storybook/main.ts

```diff
@@ -0,0 +1,24 @@
+const path = require('path');
+
+module.exports = {
+  stories: ['../src/stories/**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: ['@storybook/addon-links', '@storybook/addon-essentials'],
+  framework: '@storybook/react',
+  core: {
+    builder: 'storybook-builder-vite',
+  },
```

Comment:
Created storybook with the vite builder. This vite builder makes Storybook 10X faster.